frontend/test/lit/test_instrumentation_console.py

```diff
@@ -79,6 +75,11 @@ def circuit(weights):
 # CHECK-NEXT: [DIAGNOSTICS] Running device_release
 # CHECK-SAME:   walltime: {{[0-9\.]+}} ms{{\s*}} cputime: {{[0-9\.]+}} ms
 # CHECK:      [DIAGNOSTICS] > Total run
+# COM: As the below output is generated by catalyst-cli checking the corect order may cause falky results.
+# CHECK: [DIAGNOSTICS] Running parseMLIRSource
+# CHECK-SAME:   walltime: {{[0-9\.]+}} ms{{\s*}} cputime: {{[0-9\.]+}} ms{{\s*}} programsize: {{[0-9]+}} lines
+# CHECK: [DIAGNOSTICS] Running {{[a-zA-Z]+}}Pass
+# CHECK-SAME:   walltime: {{[0-9\.]+}} ms{{\s*}} cputime: {{[0-9\.]+}} ms{{\s*}} programsize: {{[0-9]+}} lines
```

Comment:
Does that mean the output will now show out of order for the user or is this a rare occurrence?

Reply:
no It shows fine for user, but for some reason when piping the output through FileCHECK it messes up the order.